src/services/item/plugins/app/index.ts

```diff
@@ -27,73 +27,17 @@ const plugin: FastifyPluginAsync<AppsPluginOptions> = async (fastify, options) =
   }
 
   const {
-    verifyBearerAuth,
     items: { service: itemService, extendCreateSchema, extendExtrasUpdateSchema },
   } = fastify;
 
-  if (!verifyBearerAuth) {
-    throw new Error('verifyBearerAuth is not defined!');
-  }
-
   // "install" custom schema for validating document items creation
   extendCreateSchema(createSchema);
   // "install" custom schema for validating document items update
   extendExtrasUpdateSchema(updateSchema);
 
   fastify.addSchema(common);
 
-  // register auth plugin
-  // jwt plugin to manipulate jwt token
-  await fastify.register(fastifyJwt, { secret: jwtSecret });
-
-  const promisifiedJwtSign = promisify<{ sub: AuthTokenSubject }, { expiresIn: string }, string>(
-    fastify.jwt.sign,
-  );
-
-  const aS = new AppService(itemService, jwtExpiration, promisifiedJwtSign);
-
-  const promisifiedJwtVerify = promisify<string, { sub: AuthTokenSubject }>(fastify.jwt.verify);
-
-  const validateApiAccessToken = async (jwtToken: string, request: FastifyRequest) => {
-    // try {
-    // verify token and extract its data
-    const { sub } = await promisifiedJwtVerify(jwtToken);
-
-    // TODO: check if origin in token matches request's origin ?
-    // (Origin header is only present in CORS request: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin)
-    request.authTokenSubject = sub;
-    return true;
-    // } catch (error) {
-    //   // const { log } = request;
-    //   // log.warn('Invalid app api access token');
-    //   return false;
-    // }
-  };
-
-  // bearer token plugin to read and validate token in Bearer header
-  /**
-   * This is done for performance reasons:
-   * 1. First decorateRequest with the empty type of the value to be set (null for an object)
-   *    BUT NEVER SET THE ACTUAL OBJECT IN decorateRequest FOR SECURITY (reference is shared)
-   * 2. Then later use a hook such as preHandler or onRequest to store the actual value
-   *    (it will be properly encapsulated)
-   * @example
-   *  fastify.decorateRequest('user', null) // <-- must use null here if user will be an object
-   *  // later in the code
-   *  fastify.addHook('preHandler', (request) => {
-   *     request.user = { name: 'John Doe' } // <-- must set the actual object here
-   *  })
-   * @see
-   *  https://www.fastify.io/docs/latest/Reference/Decorators/#decoraterequestname-value-dependencies
-   *  https://www.fastify.io/docs/latest/Reference/Decorators/
-   */
-  fastify.decorateRequest('authTokenSubject', null);
-
-  fastify.register(fastifyBearerAuth, {
-    addHook: false,
-    keys: new Set<string>(),
-    auth: validateApiAccessToken,
-  });
+  const aS = new AppService(itemService, jwtExpiration);
```

Comment:
I know that's not you, but I think we could rename it ^^.

```suggestion
  const appService = new AppService(itemService, jwtExpiration);
```